vault-jwt/main.tf

```diff
@@ -20,6 +20,13 @@ variable "vault_addr" {
   description = "The address of the Vault server."
 }
 
+variable "vault_jwt_token" {
+  type        = string
+  description = "The JWT token used for authentication with Vault."
+  default     = null
+  sensitive   = true
+}
+
```

Comment:
Can you add an example use case on how you would provide that token to the module? Are you fetching it externally through some API or another provider?

Reply:
> do you want the examples added to the commit in some way?
i'm unsure how documentation works here, but i'll do whatever is needed

We usually add example Terraform snippets in the `README.md` to guide users on possible ways the module can be used.

Reply:
Thanks. I would appreciate it if we could also add a vault policy that can use this new jwt token, scoped to workspace metadata.